Source/Utils/Interpolate.cpp

```diff
@@ -17,7 +17,7 @@ namespace Interpolate
         interpolated_F->setVal(0.);
 
         // Loop through the boxes and interpolate the values from the _cp data
-#ifdef _OPENMP
+#ifdef AMREX_USE_OMP
 #pragma omp parallel
```

Comment:
Missing `if (amrex::Gpu::notInLaunchRegion())`?

Reply:
I agree with you.

Reply:
```suggestion
#pragma omp parallel if (amrex::Gpu::notInLaunchRegion())
```